Description: Remove offline option from meson build
 Required for all dependencies to get pulled from cargo
 .
 scx (1.0.4.1-1ubuntu1) noble; urgency=medium
 .
   * Miscellaneous Ubuntu changes:
     - Rebase on top of the latest upstream main branch
Author: Mitchell Augustin <mitchell.augustin@canonical.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2024-09-23

--- scx-1.0.4.1.orig/meson.build
+++ scx-1.0.4.1/meson.build
@@ -276,9 +276,9 @@ if get_option('buildtype') == 'release'
   cargo_build_args += '--release'
 endif
 
-if get_option('offline')
-  cargo_build_args += '--offline'
-endif
+#if get_option('offline')
+#  cargo_build_args += '--offline'
+#endif
 
 if get_option('kernel') != ''
   kernel = get_option('kernel')
